update `installing-java.md` for prism 9.0+ (#673)
PrismLauncher/prismlauncher.org: src/wiki/getting-started/installing-java.md

  parent: Getting Started
  order: 2
---
## Automatically installing Java (recommended)

> In PrismLauncher 9.0+, you can simply let the launcher handle Java automatically on Windows and MacOS by ticking "Autodetect Java versin" and "Auto-download Mojang Java" in:

Reviewer: ```suggestion
In PrismLauncher 9.0+, you can simply let the launcher handle Java automatically on Windows, macOS and Linux (not available in all packages) by ticking "Autodetect Java version" and "Auto-download Mojang Java" in:
```

> Prism's main screen > Settings (top toolbar) > Java

This will make Prism automatically download the right Java version depending on your Minecraft version (Java 8 for pre-1.17, Java 17 for 1.17-1.20.4, and Java 21 for 1.20.5+). The rest of this page is going to explain downloading Java manually, and is probably not needed for you.

> 

Reviewer: ```suggestion

If the Java downloader is not available for you - which is the case on most Linux packages apart from the official Flatpak and AppImage downloads - manual installation is recommended instead.

```

---
# Installing Java

## Automatically installing Java (recommended)

> In PrismLauncher 9.0+, you can simply let the launcher handle Java automatically on Windows, macOS and Linux (not available in all packages) by ticking "Autodetect Java version" and "Auto-download Mojang Java" in:

Reviewer 2: ```suggestion
In Prism Launcher 9.0+, you can let the launcher handle Java automatically on Windows, macOS and Linux (not available in all packages) by enabling "Autodetect Java version" and "Auto-download Mojang Java" in:
```

> > Prism's main screen > Settings (top toolbar) > Java

Reviewer 2: ```suggestion
> Settings (top toolbar) > Java
```
Nit, but this is more in-line with other navigation instructions we have (as we can pretty safely assume most people will be on the main screen)

> This will make Prism automatically download the right Java version depending on your Minecraft version (Java 8 for pre-1.17, Java 17 for 1.17-1.20.4, and Java 21 for 1.20.5+). The rest of this page is going to explain downloading Java manually, and is probably not needed for you.

Reviewer 2: ```suggestion
This will make the launcher automatically download the right version of Java for your version of Minecraft (Java 8 for pre-1.17, Java 17 for 1.17-1.20.4, etc.). The rest of this page is going to explain downloading Java manually, and is probably not needed for you.
```

sudo apt-get update && sudo apt-get install temurin-8-jdk
```

#### Alpine Linux

```bash
sudo apk add openjdk17 openjdk8
```

#### Flatpak

The Prism Launcher Flatpak already bundles Java.

#### NixOS

The Prism Launcher NixOS package already bundles Java.

### Special cases

#### Forge 1.16.5 and Java 8u321+

Old versions of Forge crash with Java 8u321+. For this reason, using Java 8u312 or lower is recommended.

#### A note about Intel HD 2000/3000 on Windows 10

Since those iGPUs are not *officially* supported on Windows 10, with them the game is likely going to crash with any modern java binary.

> *For 1.16.5 or older* there's a workaround, you need to install an older Java binary. To do that, you should be able to simply use Mojang Java 8u51, see [above](https://prismlauncher.org/wiki/getting-started/installing-java/#automatically-installing-java-(recommended)). This should fix your issue.

Reviewer 2: ```suggestion
*For 1.16.5 or older* there's a workaround in installing an older Java binary. Mojang Java 8u51 is recommended, and can be automatically installed with the [above](#automatically-installing-java).
```
